mapiproxy/servers/default/emsmdb/dcesrv_exchange_emsmdb.c

```diff
@@ -256,23 +226,6 @@ static enum MAPISTATUS dcesrv_EcDoDisconnect(struct dcesrv_call_state *dce_call,
 		return MAPI_E_LOGON_FAILED;
 	}
 
-	/* Step 1. Retrieve handle and free if emsmdbp context and session are available */
```

Comment:
In an explicit disconnect operation sent by the client I think we should still release the session as usual